composer.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "jenyak/i18n-routing-service-provider",
+  "name": "thegeekrv/i18n-routing-service-provider",
   "description": "Silex i18n routing service provider.",
   "keywords": ["silex", "i18n routing"],
   "license": "MIT",
   "require": {
-    "silex/silex": "~2.0@dev",
+    "silex/silex": "~1.0@dev",
```

Comment:
There is another branch https://github.com/jenyak/I18nRoutingServiceProvider/tree/1.0 for support silex 1